### PR TITLE
[BLOG] Add CuPy CZI grant year 1 report blog post

### PR DESCRIPTION
**Pull request by andfoy**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] Blog post featured image is in PNG or JPEG format, **not** SVG.
- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (kgryte) [inline apps/labs/posts/cupy-czi-grant-year1.md:44]: Is it necessary to set two different seeds here?

Comment (andfoy) [reply apps/labs/posts/cupy-czi-grant-year1.md:44]: If I used the same seed in both the global and the particular generators, the signals produced don't have much variety, hence the two seeds

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (rgommers) [inline apps/labs/posts/cupy-czi-grant-year1.md:60]: This takes up a full page on my 32" monitor, as do the next figures. It would look better if it were 1.5x or 2x smaller - could you try and shrink the size a bit?

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (rgommers) [inline apps/labs/posts/cupy-czi-grant-year1.md:111]: Can you link to the actual blog post from the RAPIDS team?

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (rgommers) [inline apps/labs/posts/cupy-czi-grant-year1.md:117]: This looks slightly off with CZI being a link and Preferred Networks and following entities not. I'd suggest making them all links, or none.

Also, this is the Labs blog, so no need to thank ourselves here I'd say? The mention in the next paragraph is probably enough.

**Review by rgommers — COMMENTED** [review]
This reads quite well, nice work Edgar! A few initial comments. Also, can you please line break the content at 80 characters or so? That would make copy-edits / line-by-line review a lot easier.

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (andfoy) [reply apps/labs/posts/cupy-czi-grant-year1.md:111]: I think they haven't published it yet, that's why I left the link to their overall blog instead to the particular page, since it does not exist yet. Let me confirm with @awthomp if the blog post is now available

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (kgryte) [reply apps/labs/posts/cupy-czi-grant-year1.md:60]: Shouldn't the figure sizes be adjusted via CSS, not by literally making them smaller? Typically you want images as high-resolution as possible, otherwise you get aliasing on high resolution monitors.

Comment (rgommers) [reply apps/labs/posts/cupy-czi-grant-year1.md:60]: I meant "visually smaller". And sure, using CSS and reasonable file sizes for PNG/JPG images makes sense.

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (rgommers) [inline apps/labs/posts/cupy-czi-grant-year1.md:207]:
```suggestion
This work would not have been possible without the help of the
```

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (rgommers) [inline apps/labs/posts/cupy-czi-grant-year1.md:209]:
```suggestion
initiative, who agreed to migrate the implementations of over 36 APIs from their project to
```

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (rgommers) [inline apps/labs/posts/cupy-czi-grant-year1.md:230]:
```suggestion
where you can find more information on and discussions about implementations,
```

**Review by rgommers — COMMENTED** [review]
This reads really well. I added some minor textual suggestions.

The images look much better visually now, at 70% size. It'd be great to re-save them to make them smaller - they're very large right now which makes the whole page slow to load. I think the `dpi` keyword to Matplotlib's `savefig` function will help here - IIRC it defaults to 300 (for paper/pdf publishing), while 160 is a more normal value for the web.

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (rgommers) [inline apps/labs/posts/cupy-czi-grant-year1.md:3]: This date should be updated right before merging the post.

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (rgommers) [reply apps/labs/posts/cupy-czi-grant-year1.md]: It isn't there yet. @awthomp do you know if is it expected to arrive shortly? If so, we can wait for it - or we can just leave out this sentence, and add it back in if and when that post appears. Right now the reader is invited to go look for something that isn't there, which would be good to avoid.

**Comment by rgommers** [comment]
All LGTM now, modulo the blog link.

File: apps/labs/posts/cupy-czi-grant-year1.md
Comment (andfoy) [reply apps/labs/posts/cupy-czi-grant-year1.md]: I'll remove the sentence for the sake of publishing

**Review by rgommers — APPROVED** [review]
All looks great, let's get this in. Thanks Edgar!

**Comment by rgommers** [comment]
Oops, I forgot about Tania's comment about the added categories. Those are not just tags but add more - hence I'll open a follow-up PR to remove `CuPy` as a separate category.

**Pull request merged**
